### ISSUE TITLE
unikube deck install chooser option count

### ISSUE THREAD

**Issue by tobifroe** [unikubehq/cli#247]
Of 160 created decks, only 30 are visible in the dropdown.

**Comment by schwobaseggl** [comment]
Is the 30 a hard ceiling or dependent on your shell or screen size? Can the other decks be reached by typing and narrowing down?
30 decks in a single project is also quite a lot I think and we are trying to limit deck options that are offered the upped (or context) project anyway (see #167).

**Comment by tobifroe** [comment]
30 seems to be a hard ceiling.
![Bildschirmfoto 2021-12-10 um 09 55 17](https://user-images.githubusercontent.com/40638719/145546183-0321ae2e-c39e-43ba-b79d-a3de8d6f57de.png)
Decks that exist in a project can't be reached via filtering:
![Bildschirmfoto 2021-12-10 um 09 56 48](https://user-images.githubusercontent.com/40638719/145546397-3a8d3e64-4f98-426d-a27b-d22f87966d90.png)

![Bildschirmfoto 2021-12-10 um 09 56 25](https://user-images.githubusercontent.com/40638719/145546371-2a43cb2a-6e67-48fe-90f6-5a1fb83a1b34.png)

While 30+ decks in a single project definitely seems a lot, without a context set decks from all projects are displayed in the "dropdown", and having 6 projects with 5-7 decks seems like a plausible use case to me.

**Comment by schwobaseggl** [comment]
@tobifroe I agree that a hard ceiling is an issue in general. Having that number of projects available is one thing but having more than 1 or 2 projects up at the same time seems less likely. I think that should be the main goals:
1. restrict deck options to projects with running clusters (otherwise they cannot be installed anyway)
2. Make sure all of the decks are available (e.g. case where there are more than 30 decks in one project; If we have users with such projects, I think we can work on this full time :smile: )
